README.md

```diff
@@ -30,7 +30,7 @@ objects:
   data:
     hazelcast.xml: |-
```

Comment:
What about removing inlined xml and use directly yaml support?

Reply:
We need to wait for the 3.12 release. Then, I'll update Helm Charts and all README.